core/shared/src/main/scala/io/udash/properties/seq/CombinedReadableSeqProperty.scala

```diff
@@ -0,0 +1,68 @@
+package io.udash.properties.seq
+
+import io.udash.properties.{CrossCollections, PropertyCreator}
+import io.udash.properties.single.{CombinedProperty, ReadableProperty}
+import io.udash.utils.Registration
+
+import scala.collection.mutable
+
+class CombinedReadableSeqProperty[A, B, R: PropertyCreator](
+  s: ReadableSeqProperty[A, _ <: ReadableProperty[A]], p: ReadableProperty[B],
+  combiner: (A, B) => R
+) extends CombinedProperty[Seq[A], B, Seq[R]](s, p, null, (x, y) => x.map(v => combiner(v, y)))
+  with AbstractReadableSeqProperty[R, ReadableProperty[R]] {
+
+  protected var combinedChildren: Option[mutable.Buffer[ReadableProperty[R]]] = None
+  protected var originListenerRegistration: Registration = _
+
+  protected def originStructureListener(originPatch: Patch[ReadableProperty[A]]): Unit = {
+    val combinedNewChildren = originPatch.added.map(sub => sub.combine(p)(combiner))
+    val mappedPatch: Patch[ReadableProperty[R]] = originPatch.copy(
+      removed = originPatch.removed.indices.map(idx => combinedChildren.get(idx + originPatch.idx)),
+      added = combinedNewChildren
+    )
+    CrossCollections.replace(combinedChildren.get, originPatch.idx, originPatch.removed.size, combinedNewChildren:_*)
```

Comment:
Missing space before varargs expansion

Reply:
Did you mean: `:_*` -> `: _*`?

Reply:
Yeah, that's what IntelliJ would do when reformatting

Reply:
You saved my life.

Reply:
Always there to safeguard you from eating spaces :rescue_worker_helmet: